### PR TITLE
Update forge.yml to use local file-to-resource-mapping.json

### PR DESCRIPTION
**Pull request by MFSY**

### CODE REVIEW

File: docs/src/main/paradox/docs/getting-started/notebooks/forge.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ Store:
     project_property: "https://bluebrain.github.io/nexus/vocabulary/project"
   max_connection: 5
   versioned_id_template: "{x.id}?rev={x._store_metadata._rev}"
-  file_resource_mapping: https://raw.githubusercontent.com/BlueBrain/nexus-forge/master/examples/configurations/nexus-store/file-to-resource-mapping.hjson
+  file_resource_mapping: https://raw.githubusercontent.com/BlueBrain/nexus/8851236f5bc38e113f1f04af2ca0bb31dab1c576/docs/src/main/paradox/docs/getting-started/notebooks/file-to-resource-mapping.hjson
```
Comment (ssssarah) [inline docs/src/main/paradox/docs/getting-started/notebooks/forge.yml:23]: Why is it not https://raw.githubusercontent.com/BlueBrain/nexus/master/docs/src/main/paradox/docs/getting-started/notebooks/file-to-resource-mapping.hjson ?

Comment (MFSY) [reply docs/src/main/paradox/docs/getting-started/notebooks/forge.yml:23]: I just follow the idea of having a specific commit in ref.  I will not revise that in this PR.

Comment (ssssarah) [reply docs/src/main/paradox/docs/getting-started/notebooks/forge.yml:23]: if someone were to edit it hoping to have the edits reflected in its usage, it wouldn't work though?

Comment (MFSY) [reply docs/src/main/paradox/docs/getting-started/notebooks/forge.yml:23]: Notebooks have to be updated as well for now. That would be my next PR.

**Review by ssssarah — APPROVED** [review]
Not a fan of the commit hash but I guess if this is urgent :)

**Review by crisely09 — APPROVED** [review]

**Pull request merged**